pkg/agent/openflow/multicluster.go

```diff
@@ -25,6 +25,11 @@ import (
 // for cross-cluster traffic to distinguish from in-cluster traffic.
 var GlobalVirtualMACForMulticluster, _ = net.ParseMAC("aa:bb:cc:dd:ee:f0")
 
+// UnknownLabelIdentity represents an unknown label identity.
+// 24 bits in VNI are used for label identity. The max value is reserved for
+// UnknownLabelIdentity.
+const UnknownLabelIdentity = uint32(16777215)
```

Comment:
would be easier to understand if this is written in hexadecimal format

Reply:
Done.